foundation/foundation.css

```diff
@@ -0,0 +1,133 @@
+body {
+    font-family: Arial,Helvetica,Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 22px;
+    color: #666666;
```

Comment:
I know that we only save 300 bytes if we use #666 instead of #666666;, so i'm not so sure if we should change this or not, i let someone else says something.

Reply:
Some browsers present interpetation errors with # of only 3 numbers